src/Msg.h

```diff
@@ -160,10 +149,10 @@ class Msg {
       delete var.kafkaw_msg;
       break;
     case MsgType::Owned:
-      delete var.owned;
+      delete[] var.owned;
```

Comment:
`var.owned` was just initialised as a raw pointer, so this isn't paired with `new[]`. I think it was correct before.

Reply:
`var.owned` is being set on line 31, the actual line is:
```c++
msg.var.owned = new char[len];
```